src/pocketmine/plugin/ScriptPluginLoader.php

```diff
@@ -97,7 +97,10 @@ public function getPluginDescription($file){
 			if(preg_match("/^[ \t]+\\*[ \t]+@([a-zA-Z]+)[ \t]+(.*)$/", $line, $matches) > 0){
 				$key = $matches[1];
 				$content = trim($matches[2]);
-
+				// Check if the user intended this to be an array...
+				if($content{0} == "[" and substr($content,-1)){
```

Comment:
I think you forgot something here: `substr($content,-1)`, wouldn't that be `substr($content,-1) === "]"` (or `$content{strlen($content) - 1} === "]"`)?

Also please use `===` instead of `==` in the first condition.